react-native-libraries.json

```diff
@@ -6170,5 +6170,16 @@
     "windows": false,
     "macos": false,
     "unmaintained": false
+  },
+  {
+    "githubUrl": "https://github.com/microsoft/fluentui-react-native",
+    "npmPkg": "fluentui-react-native",
```

Comment:
```suggestion
    "npmPkg": "@fluentui/react-native",
```

Reply:
thanks i change all your suggestions thank you @brentvatne !.